src/rust/engine/process_executor/src/lib.rs

```diff
@@ -69,15 +57,15 @@ pub fn run_command(req: ExecuteProcessRequest) -> Result<ExecuteProcessResult, E
 #[cfg(test)]
 mod tests {
     use super::{ExecuteProcessRequest, ExecuteProcessResult, run_command};
-    use std::collections::HashMap;
+    use std::collections::BTreeMap;
 
     #[test]
     #[cfg(unix)]
     fn stdout() {
-        let result = run_command(ExecuteProcessRequest::new(
-            make_argv(&["/bin/echo", "-n", "foo"]),
-            HashMap::new(),
-        ));
+        let result = run_command(ExecuteProcessRequest {
+            argv: make_argv( & ["/bin/echo", "-n", "foo"]),
```

Comment:
the (non) spacing around `&` on the LHS was "correct" and also consistent with your other `make_argv` calls.

Reply:
I have no idea how this happened. Done.

I'll also reformat this before merging, because it looks like the rest of the engine uses two spaces not four.